packages/cardpay-sdk/index.ts

```diff
@@ -74,5 +74,51 @@ export {
   type SchedulePaymentProgressListener,
 } from './sdk/scheduled-payment-module';
 export { default as ScheduledPaymentModuleABI } from './contracts/abi/modules/scheduled-payment-module';
-export * as ClaimSettlement from './sdk/claim-settlement/utils';
+export {
+  Address,
+  TransferERC20ToCaller,
+  NFTOwner,
+  TransferNFTToCaller,
+  Claim,
+  getTypedData,
+} from './sdk/claim-settlement/utils';
 export { default as ClaimSettlementModule } from './sdk/claim-settlement-module';
+
+// add these classes for typedoc to display documentation
```

Comment:
We only get docs from things which are exported. I tried not to do this too much but the tradeoff here is if we want better links/references to our doc, we should export them. Thats why typedoc tends to work well with packages we use as "libraries" not "services"